File: dayTwo/Javascript/QuestionThree.js
let sum = 0;

for(let count = 1; count <= 100; count++){
    let number = count;
    while(number > 0){
        let digit = number % 10;
        sum += digit;
        number = Math.floor(number / 10);
    }
    if(sum % 3 != 0){
        process.stdout.write(count + " ");
    }
}

File: dayTwo/Java/QuestionThree.java
public class QuestionThree {
    public static void main(String... args){
        
        int sum = 0;
        
        for(int count = 1; count <= 100; count++){
            sum = 0;
            int number = count;
            while(number > 0){
                int digit = number % 10;
                sum += digit;
                number /= 10;
            }
            if(sum % 3 != 0){
                System.out.print(count + " ");
            }
        }
    }
}
